### PR TITLE
Cargo sloths

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Adds some more theme ruining bloat.

https://file.house/v4C3.dmi sprites are there

:cl: Kor
add: Added sloths to cargobay on Box and Meta
/:cl:

### CODE REVIEW

**Comment by chanoc1** [comment]
do the sloths have names

if they dont i hope this gets closed :^)

**Comment by KorPhaeron** [comment]
The one on Box is named Paperwork, and the one on Meta is named Citrus. They both have their own sprites.

**Comment by chanoc1** [comment]
NICE

**Comment by ChangelingRain** [comment]
sprite issues sprite issues aaaaah;
the base sloth sprite has the west sprite just as a copy of the north sprite
all the east/west sloth sprites except for that have a bunch of weirdly-placed 2 alpha pixels
the horrifying skull-like face of the sloth will haunt your dreams

**Comment by FantasticFwoosh** [comment]
Its a pity they dont have their own little yellow cargo uniforms.

**Comment by optimumtact** [comment]
@ChangelingRain inside every sloth is a skeleton waiting to burst out

**Comment by KorPhaeron** [comment]
>the horrifying skull-like face of the sloth will haunt your dreams

sloth

**Comment by Bawhoppen** [comment]
Do they move ultra slow but then when theres no gravity they get to be the speeddters they dreamt of?

**Comment by RemieRichards** [comment]
WHY, I have terrible deja vu of this ENTIRE PR.
including "the horrifying skull-like face of the sloth will haunt your dreams" and that picture response...

WTF.

**Comment by KorPhaeron** [comment]
@RemieRichards 

Now that you mention it, it does seem a bit familiar. oranges used to post that picture frequently though, maybe a similar exchange happened on singulo

**Comment by RemieRichards** [comment]
No no you don't get it.
This PR, how it looks in my emails (gmail), is 100% as I see.

It's either clairvoyance or just standard deja vu (which is just the brain failing to process something in the right order, leading you to "remember" it before you've "seen" it)
but i swear it's not just deja vu it's so real fuck.

**Comment by Iamgoofball** [comment]
@RemieRichards thanks you just ruined deja vu for me

**Comment by Shadowlight213** [comment]
Remie has a 6th sense for detecting sloth prs.
We have discovered her super power. The government is inbound. o7

File: code/modules/mob/living/simple_animal/friendly/sloth.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/mob/living/simple_animal/sloth
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/friendly/sloth.dm]: why not a child of /pet ?

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/friendly/sloth.dm]: The only thing in /pet seemed to be code for adding pet collars, and I don't want to make new overlays for sloth collars.

**Comment by WJohn** [comment]
The regular sloth's west sprite is using the north sprite. Also some rogue 2 alpha pixels around the east/west sprites on both regular and cool sloths. Easy to fix.

**Comment by optimumtact** [comment]
The DMI in the OP is not the same on as in the PR anymore

**Comment by ChangelingRain** [comment]
I already commented that wj! also he still hasn't fixed the alpha pixels so it's not totally wrong

**Comment by Niknakflak** [comment]
I dont want to download the sprite, where's my preview

**Comment by Iamgoofball** [comment]
Sloth lives

**Comment by Chocobro** [comment]
i'd say that Citrus is a player reference but oranges doesn't play the game so
aytomhsere-2 1

**Comment by optimumtact** [comment]
Also my player isn't named Citrus, nor is it my byond name you clown

**Comment by Chocobro** [comment]
only oranges would get salty at such a prime rib roast

**Comment by PKPenguin321** [comment]
🔥🍊🔥

**Pull request merged**